src/Microsoft.AspNet.Razor/Parser/TagHelpers/TagHelperBlockRewriter.cs

```diff
@@ -148,13 +154,17 @@ public static class TagHelperBlockRewriter
                         symbolOffset = 0;
```

Comment:
We don't want to accept the end `"` or `'` into the newly created attribute value.